CI/dependencies_setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Base Image Updates
+sudo apt-get update -y && sudo apt-get install -y git && sudo apt-get install -y cmake && sudo apt-get install -y build-essential && sudo apt-get install -y apt-utils && sudo apt-get install -y libssl-dev && sudo apt-get install wget
```

Comment:
You can let one single line here for installing everything. Also try to split commands on different lines - it's easier to do debugging. You may have something like:

```
sudo apt-get update -y 
sudo apt-get install -y git cmake [all the packages split by space]
```